src/dashboard/components/DonationSummary.vue

```diff
@@ -127,22 +120,6 @@ export default {
         updates: { address: address },
       });
     },
-    donateAll() {
-      this.$store.dispatch('metamask/donateAll', this.distribution).catch(e => {
-        console.error('donateAll (in vue) error:', e);
-        // We're currently not catching the emitting anywhere so we
-        // console.error for now
-        this.$emit('error', e);
-      });
-    },
-    distribute() {
```

Comment:
Don't move this, it conflicts with the refactor I'm doing. My pull request (https://github.com/SuperuserLabs/thankful/pull/213) will move the distribution data to vuex store so it is accessible from anywhere.

Reply:
While unfortunate that you both poked at the same code, the easiest would probably be to just let whoever merges last to resolve the conflicts.